Assets/Scripts/Singletons/DataModelSingletons/MissionsManager.cs

```diff
@@ -117,6 +119,20 @@ private static void CompleteMission(ScheduledMission scheduled)
         ScheduledMissions.Remove(scheduled);
     }
 
+    private static void UnlockMissions()
+    {
+        foreach (Mission mission in Instance.Missions)
+        {
+            if (!mission.HasBeenUnlocked 

```

Comment:
Basically we need to unlock any missions that require less money than the player's starting money. But we also need to subscribe to the event for any locked money-requiring missions. So I just do it on Init. Could do the latter on new game start but then we're still running checks.